Tasks/UniversalPackagesV0/Common/ArtifactToolUtilities.ts

```diff
@@ -45,53 +56,43 @@ export async function getArtifactToolFromService(serviceUri: string, accessToken
     if (arch === "x64"){
         arch = "amd64";
     }
+
     const blobstoreAreaName = "clienttools";
     const blobstoreAreaId = "187ec90d-dd1e-4ec6-8c57-937d979261e5";
     const ApiVersion = "5.0-preview";
 
-    const credentialHandler = vsts.getBasicHandler("fakeUsername", accessToken);
+    const credentialHandler = vsts.getBasicHandler("vsts", accessToken);
     const blobstoreConnection = new vsts.WebApi(serviceUri, credentialHandler);
 
-    let blobstoreApi = blobstoreConnection.getCoreApi();
-
-    // Get packaging versions url
-    const artifactToolGetUrl = await new Promise<string>((resolve, reject) => {
-        let getToolDataPromise = blobstoreApi.vsoClient.getVersioningData(ApiVersion,
+    try{
+        const artifactToolGetUrl = await blobstoreConnection.vsoClient.getVersioningData(ApiVersion,
             blobstoreAreaName, blobstoreAreaId, { toolName }, {osName, arch});
-        getToolDataPromise.then((result) => {
-            return resolve(result.requestUrl);
-        });
-        getToolDataPromise.catch((error) => {
-            return reject(error);
-        });
-    });
 
-    // Get highest version
-    const artifactToolUri =  await new Promise<any>((resolve, reject) => {
-        blobstoreApi.restClient.get(artifactToolGetUrl, ApiVersion, null, { responseIsCollection: false },
-            async function (error, status, toolResult) {
-                if (!!error || status !== 200) {
-                    return reject(error.message);
-                }
-                // Getting the tool uri
-                return resolve(toolResult);
-            });
-    });
+        const artifactToolUri =  await blobstoreConnection.rest.get(artifactToolGetUrl.requestUrl);
```

Comment:
I had to change this because new vsts-node-api package that supports resource area apis had other breaking changes.